lib/kb_blast/Utils/BlastUtil.py

```diff
@@ -651,6 +651,28 @@ def write_target_obj_to_file (self, params, input_many_ref, seq_type):
                 genome_ref = input_many_genomeSet['elements'][genome_id]['ref']
                 target_feature_info['genome_id_to_genome_ref'][genome_id] = genome_ref
 
+            """
```

Comment:
debugging comment -- remove before committing.

Reply:
Generally, my policy is that if it's useful enough to keep around, turn it into a proper logging statement with log level set to 'DEBUG'. Otherwise, get rid of it -- it's adding extra cruft to the code and increasing the cognitive burden for other people who have to read through the code and work out what it is supposed to do.

Reply:
done